.env.test

```
@@ -8,8 +8,6 @@ FLASK_DEBUG=true
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
 
-# Required to interact with the Trello API
-TRELLO_API_KEY=api_key
-TRELLO_API_TOKEN=api_token
-TRELLO_BOARD_ID=board_id
-TRELLO_DEFAULT_LIST_ID=list_id
\ No newline at end of file
+# Mongo connection details
+MONGO_CONNECTION_STRING="mongodb://mongodb:27017"
```

```suggestion
MONGO_CONNECTION_STRING=mongodb://mongodb:27017
```
This will seem petty, but if you pass a this through to Docker with the `--env-file` flag, it treats the quotes as part of the actual value, and everything breaks!